components/alibi-detect-server/adserver/cm_model.py

```diff
@@ -21,7 +21,7 @@
 SELDON_PREDICTOR_ID = DEFAULT_LABELS["predictor_name"]
 
 
-def _load_class_module(module_path: str) -> str:
+def _load_class_module(module_path: str):
```

Comment:
Why? It still seems to return `str`, or not?

Reply:
It actually returns a `Module`. It was raising type hint warnings because it's used like so in L77-79

```python
            # Load from locally available models
            MetricsClass = _load_class_module(self.storage_uri)
            self.model = MetricsClass()
```

Reply:
so potentially `-> "Module"` (or other appropriate way to annotate the return type) could be good?

Reply:
probably you'd need to import appropriate type